### PR TITLE
assertion, searching and filtering were added

### PR DESCRIPTION
**Pull request by olpapina**

### CODE REVIEW

File: cypress/e2e/spec.cy.js [hunk 1]
```diff
@@ -2,6 +2,85 @@
...
+    homePage.openHomePage()
+    homePage.getSearchBox().typeSearchedProduct(testData.searchText)
+  })
+
+  it('verify results by typing text', () => {
+    const resultPage = homePage.getSearchBox().clickSearchButton()
+    resultPage.getResultBlock().getResultSearchLinks()
+      .should('exist')
+      .each(link => {
+        expect(link.text()).to.contain('iPhone')
```
Comment (twikssi) [inline cypress/e2e/spec.cy.js]: put Iphone to the file

File: cypress/e2e/spec.cy.js [hunk 2]
```diff
@@ -79,7 +158,7 @@ describe('Tests verify Gift Card Delivery according to location', () => {
     homePage.getActualDeliveryLocation().should('contain', expectedLocation)
     const giftCardPage = homePage.getMenuBar().clickGiftCardTabButton()
     const productPage = giftCardPage.selectTypeGiftCard('eGift')
-    productPage.getDeliveryType().then( value => {
+    productPage.getDeliveryType().then(value => {
       expect(value).to.equal('Email')
```
Comment (twikssi) [inline cypress/e2e/spec.cy.js]: put Email to the file

Comment (olpapina) [reply cypress/e2e/spec.cy.js]: fixed in 12fa312c

File: cypress/e2e/spec.cy.js (hunk 1 above)
Comment (olpapina) [reply cypress/e2e/spec.cy.js]: fixed in 12fa312c

**Pull request merged**